### DISCUSSION TITLE
Is the 6.1.2 Patch-T problem fixed in 7.x?

### DISCUSSION THREAD

**Discussion by borque** [cyanfish/naps2#245] [Support]
Hello!
In version 6.1.2 there was a problem with Patch-T fileseparation. It only worked under certain circumstances: https://sourceforge.net/p/naps2/discussion/general/thread/936c94f8/ .
I did not see this in the release notes for 7.x, so my question is whether this has been fixed or not.
Thanks for an answer and for this otherwise great software!

PS: The links on naps2.com still point to sourceforge for discussions. I hope it is ok to post my question here?

**Comment by cyanfish** [comment]
I think it should work, but why not try and see for yourself? If you don't want to replace the old version you can download the [portable](https://www.naps2.com/download) (.zip).
